sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerConfig.java

```diff
@@ -57,6 +57,9 @@
   @JsonProperty
   private boolean useFallback = false;
 
+  @JsonProperty
+  private boolean forceTimeCondition = false;
```

Comment:
Suggest "requireTimeCondition" as the name instead. 

"force" to me sounds like requiring the server to plan queries in a certain way, while this is more of a requirement on queries submitted by the user.